frontend/src/util/Hook.tsx

```diff
@@ -30,6 +30,68 @@ export const useBestSubmission = (player?: Player | null) => {
   return bestSubmission;
 };
 
+export const useGetScore = (player?: Player) => {
+  const counted = new Set<number>();
+  const [score, setScore] = useState<number>(0);
```

Comment:
If things are working on the backend, I actually don't think we need this at all - I think it can be replaced with something as simple as `player.solved.filter((s) => s)`

Reply:
I'm not quite sure what you mean by that. We can talk about it in the standup.

Reply:
After talking about it, I'll implement this change in my upcoming PR 👍